### PR TITLE
sync contents to latest release

### PR DESCRIPTION
**Pull request by gaohoward**
Signed-off-by: Howard Gao <howard.gao@gmail.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by gaohoward** [comment]
@goldmann Hi Marek,
This is the sync PR for openshift cp branch. Please review.
Thanks
Howard

File: image.yaml [hunk 1]
```diff
@@ -151,5 +151,5 @@ run:
           - "/opt/amq/bin/launch.sh"
 osbs:
       repository:
-            name: jboss-amq-6-docker
+            name: containers/jboss-amq-6
             branch: ce-1.3-openshift-amq-6.3-jdk-8-rhel-7
```
Comment (goldmann) [inline image.yaml]: Branch is different too: `jb-amq-6.3-openshift-rhel-7`. The suggestion is that the branch in the image image descriptor is pointing to the "main" target branch, but the specific one (`jb-amq-6.3-openshift-cp-rhel-7`) is added to an override file.

**Review by goldmann — CHANGES_REQUESTED** [review]
Please update the branch.

**Comment by gaohoward** [comment]
OK I'll change it. Btw in the actual building, is it ok just to use --override option in the command-line?
Howard

**Comment by gaohoward** [comment]
@goldmann Just changed to sprint-21.

**Review by goldmann — APPROVED** [review]

**Pull request merged**